.changeset/metal-shrimps-yawn.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/box': major
+---
+
+Box component now accepts "as" prop instead of "component" and exports generic ExtendableBox as helper type for components wrapping Box
```

Comment:
```suggestion
Box component now accepts `as` prop instead of `component` and exports generic `ExtendableBox` as helper type for components wrapping Box.

Example usage of `ExtendableBox`: ...
```

Reply:
👍